README.md

```diff
@@ -214,6 +218,13 @@ Organized by the Linux Foundation, it's a great free course to learn basics abou
 
 If you used Windows and decided to switch to Linux, this course should help you to find their alternatives. It's pretty short and you can finish it in a day. - [@aleksandar-todorovic](https://github.com/aleksandar-todorovic)
 
+### Articles
```

Comment:
Can you please change the heading to "Bash Learning Resources"?

"Articles" kind of makes it sound like there would be articles on other subjects.